lib/plugins/wallet/litecoind/litecoind.js

```diff
@@ -125,11 +125,16 @@ function checkBlockchainStatus (cryptoCode) {
     .then(res => !!res['initialblockdownload'] ? 'syncing' : 'ready')
 }
 
+function getTxHashesByAddress (cryptoCode, address) {
+  throw new Error(`Transactions hash retrieval not implemented for this coin!`)
+}
+
```

Comment:
Litecoin in theory has access to similar RPC methods as Bitcoin, should be the same implementation

```suggestion
function getTxHashesByAddress (cryptoCode, address) {
  checkCryptoCode(cryptoCode)
    .then(() => fetch('listreceivedbyaddress', [0, true, true, address]))
    .then(txsByAddress => Promise.all(_.map(id => fetch('getrawtransaction', [id]), _.flatMap(it => it.txids, txsByAddress))))
    .then(_.map(({ hash }) => hash))
}

```